pages/clustering/high-availability.mdx

```diff
@@ -128,7 +129,7 @@ export MEMGRAPH_MANAGEMENT_PORT=10011
 export MEMGRAPH_BOLT_PORT=7692
 ```
 
-When using either of these flags, configurations flags for `--bolt-port`, `--management-port` and `--experimental-enabled` will be ignored.
+When using either of these envronment variables, flags `--bolt-port`, `--management-port` and `--experimental-enabled` will be ignored.
```

Comment:
I've updated the statement to use "any" instead of "either". -> https://github.com/memgraph/documentation/pull/804/commits/0002026276044e905db75dbe5363dbc073f6b339